src/zenml/artifacts/utils.py

```diff
@@ -100,6 +101,7 @@ def save_artifact(
     is_model_artifact: bool = False,
     is_deployment_artifact: bool = False,
     manual_save: bool = True,
+    is_external_artifact: bool = False,
```

Comment:
Instead of passing `manual_save` and `is_external_artifact` here, can't we just simply pass the type already and default it to `manual`?